homeassistant/components/fan/isy994.py

```diff
@@ -93,6 +97,11 @@ def turn_off(self, **kwargs) -> None:
         else:
             self.speed = self.state
 
+    @property
+    def speed_list(self: ToggleEntity) -> list:
```

Comment:
Remove the `ToggleEntity` annotation on `self`. That's not needed and we're not strict about annotations anyway.
https://www.python.org/dev/peps/pep-0484/#annotating-instance-and-class-methods